app/helpers/date_validation_helper.rb

```diff
@@ -22,4 +13,10 @@ def start_date_before_end_date
 
     errors.add(:start_date, :before) unless start_date <= end_date
   end
```

Comment:
General observation, unrelated to this PR - `start_date_before_end_date` doesn't seem like it should live here. It's an actual validation in its own right rather than just a helper. Probably worth co-locating with the logic in DateValidator.